jwql/tests/test_database_interface.py

```diff
@@ -39,7 +42,8 @@ def test_anomaly_records():
     """Test to see that new records can be entered"""
 
     # Add some data
-    di.session.add(di.Anomaly(rootname='foo1', flag_date=datetime.datetime.today(), user='test', ghost=True))
+    random_string = ''.join(random.SystemRandom().choice(string.ascii_lowercase + string.ascii_uppercase + string.digits) for _ in range(10))
+    di.session.add(di.Anomaly(rootname=random_string, flag_date=datetime.datetime.today(), user='test', ghost=True))
     di.session.commit()
 
     # Test the ghosts column
```

Comment:
Would you need to include a filter for the rootname in the query below?

Reply:
@laurenmarietta Ah, nice catch. Yes, that would improve the accuracy of the test. 

Since we were waiting on this for #430 and $431, I propose to make that change in a future PR that also includes other improvements to this module, such as testing the other tables and connections that make up `database_interface`.  I can open an issue for that.